docs/docs/deploying-to-cloudflare-workers.md

```diff
@@ -0,0 +1,126 @@
+---
+title: Deploying to Cloudflare Workers
+---
+
+[Cloudflare Workers](https://workers.cloudflare.com/) is a serverless platform for creating entirely new applications or augmenting existing ones without configuring or maintaining infrastructure. With [Workers Sites](https://developers.cloudflare.com/workers/sites/start-from-existing/) you can deploy any static site including your Gatsby projects to a domain on Cloudflare or for free with on your [workers.dev](workers.dev) subdomain.
+
+This guide will get you started in a few steps:
+
+1. Installing wrangler CLI
+
+Workers Sites requires [wrangler](https://developers.cloudflare.com/workers/tooling/wrangler/). The more straight forward way to install wrangler is with [npm](https://www.npmjs.com/), run the following command:
+
+```shell
+npm i -g @cloudflare/wrangler
+```
+
+2. Initialize the Project
+
+To create the Worker code that will serve your Gatsby files, from the root of your Gatsby project run:
+
+```
+wrangler init --site
+```
+
+You'll notice your project structure should now look something like:
+
+```
+├── workers-site
+└── wrangler.toml
+│   ├── index.js
+│   ├── package-lock.json
+│   └── package.json
+├── public
+│   ├── 404
+│   ├── 404.html
+    ..
+├── src
+│   ├── components
+│   ├── images
+│   └── pages
+├── gatsby-config.js
+├── package.json
+```
+
+3. Configure
+
+To authenticate into your Cloudflare account run:
+
+```
+wrangler config
+```
+
+Follow the [Quick Start](https://developers.cloudflare.com/workers/quickstart/#configure) for steps on gathering the correct account ID and API token to link wrangler to your Cloudflare account.
+
+If you don't already have a workers.dev domain run:
+
+```
+wrangler subdomain
+```
+
+Then, add to your `wrangler.toml` the account ID and set `bucket` to the Gatsby files:
+
+```
+name = "gatsby-project"
+type = "webpack"
+account_id = "abcd..."
+workers_dev = true
+
+[site]
+bucket = "./public"
+entry-point = "workers-site"
+```
+
+This deploys to your workers.dev subdomain. For a custom domain see [Quick Start](https://developers.cloudflare.com/workers/quickstart/#publish-to-your-domain).
+
+4. Deploy
+
+You can deploy your application by running the following command in the root of the project directory:
+
+```shell
+wrangler publish
```

Comment:
I got to this point okay and get the following error:

```
Error: You will need to enable Workers Unlimited for your account before you can use this feature.
```

Looking into this it seems like the `public` folder gets saved to Cloudflare's KV storage which requires a premium plan, is that correct? I didn't have one, maybe we can make it clear at the top of the article that you need a premium account? Perhaps I'm mistaken and there is a way to do this on a free tier but I haven't been able to find it.

Reply:
Done